src/battle_script_commands.c

```diff
@@ -9272,6 +9285,89 @@ static void Cmd_various(void)
             gBattlescriptCurrInstr += 7;
         return;
     }
+    case VARIOUS_JUMP_IF_FLING_FAILS:
+    #ifdef ITEM_EXPANSION
+        if (!ItemId_GetFlingPower(gBattleMons[gActiveBattler].item))
+            gBattlescriptCurrInstr = T1_READ_PTR(gBattlescriptCurrInstr + 3);
+        else if (gLastUsedItem == ITEM_NONE)
+    #else
+        if (gLastUsedItem == ITEM_NONE)
+    #endif
```

Comment:
IMO, `ItemId_GetFlingPower` and `flingPower`should be added by both BE and IE. That way, we allow people who only use BE to also be able to use Fling (they would only need to add the data to `gItems`.

Reply:
I don't agree with this because the functions are item specific as I mentioned on Discord, but also now that I'm checking, the changes are bound to cause at least 2 merge conflicts *(`include/item.h` and `src/item.c`)* due to the placement of `ItemId_GetFlingPower` and the existence of `IsPinchBerryItemEffect` in the battle_engine, which does not exist in the item_expansion.

Because of that, I rather wait for a second opinion :eyes: